lib/adequate_serializer/controller.rb

```diff
@@ -0,0 +1,40 @@
+require 'adequate_serializer/helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.